islandora_bookmark.module

```diff
@@ -662,7 +668,7 @@ function islandora_bookmark_detailed_form_manage(array $form, array &$form_state
   $bookmark = Bookmark::getList($list_id);
 
   // Manage.
-  if ($bookmark->bookmarkOwner === $user->uid) {
+  if ($bookmark->bookmarkOwner === $user->uid && $bookmark->getIsDeletable()) {
```

Comment:
@jordandukart will look at it tomorrow. I have some new code to pull in here to address @jonathangreen concerns. Also, some of this is a bit old and needs to brought up to date.